src/components/AddHouseForm/AddHouseForm.jsx

```diff
@@ -0,0 +1,165 @@
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Checkbox from '@mui/material/Checkbox';
+import FormControl from '@mui/material/FormControl';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormGroup from '@mui/material/FormGroup';
+import InputAdornment from '@mui/material/InputAdornment';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import { useState } from 'react';
+
+import styles from './AddHouseForm.module.css';
+
+const AddHouseForm = () => {
+  const [moreFacilities, setMoreFacilities] = useState(false);
```

Comment:
As this has to do more with displaying more facilities than setting them and the value of the state is boolean, I'd suggest a slight name change to `moreFacilitiesShown`/`setMoreFacilitiesShown`.